frontend/src/components/EmptyCanvas.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+export default function EmptyCanvas() {
```

Comment:
When should this appear? Only when there are no Viewers or also when we have Viewers with 0 instances?
TODO: Also redux logic should remove it on AddViewer action